packages/filebrowser-extension/src/index.ts

```diff
@@ -1172,6 +1175,22 @@ function addCommands(
     }
   });
 
+  commands.addCommand(CommandIDs.toggleFileCheckboxes, {
+    label: trans.__('Show File Checkboxes'),
+    isToggled: () => browser.showFileCheckboxes,
+    execute: () => {
+      const value = !browser.showFileCheckboxes;
+      const key = 'showFileCheckboxes';
+      if (settingRegistry) {
+        return settingRegistry
+          .set('@jupyterlab/filebrowser-extension:browser', key, value)
```

Comment:
Optional for this PR: we could take the opportunity to hoist `@jupyterlab/filebrowser-extension:browser` in a `const FILE_BROWSER_PLUGIN_ID` variable at the module level, since this seems to be used in several places now.